src/main/java/org/example/repository/RealtyAgentRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.example.repository;
+
+import org.example.entity.RealtyAgent;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface RealtyAgentRepository extends JpaRepository<RealtyAgent, Integer> {
+
+    RealtyAgent findRealtyAgentById(int id);
```

Comment:
You cour remove this method from repository and use standart `findById` from JpaRepository